contracts/vault/packages/VaultPackage.sol

```diff
@@ -307,7 +314,7 @@ contract VaultPackage is VaultStorage, IVault, IVaultInit, IVaultEvents {
     /// @param newDebt The target debt for the strategy.
     /// @return The amount of debt added or removed.
     // solhint-disable-next-line function-max-lines,code-complexity
-    function updateDebt(address strategy, uint256 newDebt) external override onlyRole(STRATEGY_MANAGER) nonReentrant returns (uint256) {
+    function updateDebt(address strategy, uint256 newDebt) external override onlyManager(msg.sender) nonReentrant returns (uint256) {
```

Comment:
no need to pass msg.sender as a parameter, modifiers have access to msg data. You can make it just "onlyManager"
modifier in this case will look like this:

```
modifier onlyManager(address account) {
        if (!hasRole(STRATEGY_MANAGER, msg.sender) || !hasRole(DEBT_MANAGER, msg.sender)) {
            revert NotAuthorized();
        }
        _;
    }
```